### PR TITLE
Update maintainers list

### PR DESCRIPTION
**Pull request by msfussell**
Updating the list of maintainers

### CODE REVIEW

**Comment by msfussell** [comment]
Once this is change is approved the following updates are made to complete the maintainer process

1) CNCF Dapr Maintainers list updated here https://github.com/cncf/foundation/blob/main/project-maintainers.csv#L979
2) Email addresses are update in https://lists.cncf.io/g/cncf-dapr-maintainers/directory by emailing ascavarda@linuxfoundation.org with the changes.
3) The maintainer teams are updated in the Dapr org https://github.com/orgs/dapr/teams?query=maintainer

**Review by greenie-msft — APPROVED** [review]

**Review by yaron2 — APPROVED** [review]

**Pull request merged**
